src/main/java/org/jenkinsci/plugins/workflow/cps/DSL.java

```diff
@@ -247,21 +262,6 @@ protected Object invokeStep(StepDescriptor d, String name, Object args) {
             d.checkContextAvailability(context);
             Thread.currentThread().setContextClassLoader(CpsVmExecutorService.ORIGINAL_CONTEXT_CLASS_LOADER.get());
```

Comment:
Things That Go Bump In The Night: changing the classloader used to run code -- which is a tongue-in-cheek way of saying "there may be a monster here, but I'm not 100% sure."

Reply:
No, this call has test coverage IIRC. Anyway it is not related to this PR.

Reply:
Never mind, retracted, I missed something.